packages/plans-grid-next/src/components/features-grid/index.tsx

```diff
@@ -217,23 +220,24 @@ const TabletView = ( {
 };
 
 const FeaturesGrid = ( {
-	gridPlans,
+	currentSitePlanSlug,
```

Comment:
**Note:**

The only meaningful change to props here is the addition of `showRefundPeriod`. The rest of the changes were me sorting the props in alphabetical order as is already done elsewhere in this file.